config/common/dev.common.config.js

```diff
@@ -0,0 +1,132 @@
+const HtmlWebPackPlugin = require("html-webpack-plugin");
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const WebfontPlugin = require("webfont-webpack-plugin").default;
+const CopyPlugin = require('copy-webpack-plugin');
+const webfontConfig = require('./webfont.config')
+const lessConfig = require('./less.config')
+const scssConfig = require('./scss.config')
+const path = require('path')
+const fs = require('fs')
+const pkg = require('../../package.json')
+
+const product_list = [
+  'summernote',
+  'summernote-bs4',
+  'summernote-lite'
+]
+
+let entries = {} 
+fs.readdirSync(path.resolve(__dirname, '../../lang')).forEach(file => {
+  const filename = file.replace('.js', '');
+  entries[filename] = `./lang/${filename}`
+})
+
+module.exports = function (product) {
+  
+  return {
+    entries,
+    output: {
+      path: path.join(__dirname, "dist"),
+      filename: (chunkData) => {
+        
+        var isProduct = product_list.includes(chunkData.chunk.name)
+
+        return isProduct ? '[name].js': 'lang/[name].js';
+      }
+    },
+    externals: {
+      jquery: "jQuery"
```

Comment:
I noticed this script using `"` and `'` mixed. How about including this file into lint?